templates/media/album-gallery-item.php

```diff
@@ -26,7 +26,9 @@
 	$rtmedia_backbone['is_edit_allowed'] = $is_edit_allowed[0];
 }
 ?>
-<li class="rtmedia-list-item" id="<?php echo esc_attr( rtmedia_id() ); ?>">
+
+<?php // rtmedia_id is return backbone object if we use esc_attr then it will create > & < into &lt; &gt; it will not interpret backbone object into media id. ?>
+<li class="rtmedia-list-item" id="<?php echo rtmedia_id(); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>">
```

Comment:
Here we can use `esc_attr` function instead of phpcs:ignore

Reply:
rtmedia_id is return backbone object if we use esc_attr then it will create > & < into &lt; &gt; it will not interpret backbone object into media id.
i didn't do it because of this comment.